app/src/views/private/components/refresh-sidebar-detail.vue

```diff
@@ -35,16 +36,28 @@ export default defineComponent({
 
 		const activeInterval = ref<NodeJS.Timeout | null>(null);
 
+		const setRefreshInterval = (interval: number) => {
+			activeInterval.value = setInterval(() => {
+				emit('refresh');
+			}, interval * 1000);
+		};
+
+		idleTracker.on('hide', () => {
```

Comment:
Should we also listen to the `idle` event?

https://github.com/directus/directus/blob/cb275308eb7f2a7490ad8b06e1c2775c94edfcef/app/src/auth.ts#L62-L63

Reply:
Good catch! 👍 We should, as the token refresh stops when `idle`.

Reply:
> 

Done 👍

Reply:
Nice, thanks! 🥇